minio/operator
# Move Bucekt DNS feature to sidecar

Moves the Bucket DNS webhook from Operator to the sidecar along with the Tenant, no longer needing a tenant to reach to operator for Bucket Service creation.

# Testing this PR

1. Deploy a tenant and wait for it to come online
2. edit the tenant and add `.spec.features.bucketDNS = true`, wait for the rolling restart
3. Create a new bucket
4. verify a service with the same name of the bucket is created
